frontend/apps/docs/content/docs/parser/supported-formats/bigquery.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: BigQuery
+---
+
+Currently, BigQuery is supported through the tbls integration. While direct BigQuery support is not yet implemented, you can use tbls as a workaround to generate schema documentation for your BigQuery datasets.
+
+## Using tbls with BigQuery
+
+1. First, install tbls by following the [installation instructions](https://github.com/k1LoW/tbls?tab=readme-ov-file#install)
+
+2. Set up Google Cloud authentication:
+
+   - Create a service account and download the JSON key file
+   - Set the environment variable: `export GOOGLE_APPLICATION_CREDENTIALS="/path/to/key.json"`
+
+3. Use tbls to generate a schema.json file from your BigQuery dataset:
+
+```bash
+tbls out -t json -o schema.json "bigquery://project-id/dataset-name"
+```
+
+Replace the following with your BigQuery details:
+
+- `project-id`: Your Google Cloud project ID
+- `dataset-name`: Your BigQuery dataset name
```

Comment:
```suggestion
tbls out -t json -o schema.json "bigquery://project-id/dataset-id"
```

Replace the following with your BigQuery details:

- `project-id`: Your Google Cloud project ID
- `dataset-id`: Your BigQuery dataset ID
```